docs/modules/quickstarters/partials/secret-scanning-with-gitleaks.adoc

```diff
@@ -0,0 +1,13 @@
+== Secret scanning with gitleaks
+
+This repository has a pre-commit hook that runs gitleaks on every commit.
+To install the hook, run:
+
+```
+pip install pre-commit
```

Comment:
Let's rather link to the page Gerard already mentioned indicating that there are also 0-dependency and platform installation avenues. It looks like https://pre-commit.com is blocked internally at the moment however.